src/js/components/PackageUpgradeOverview.js

```diff
@@ -1,135 +0,0 @@
-import React from 'react';
```

Comment:
Yeah I recommended removing it because the files are not being used in master and I don't think it makes sense to maintain them. If we do merge the software update feature branch in the future, we can pick up these loose ends as needed.